.circleci/config.yml

```diff
@@ -5,23 +5,40 @@ jobs:
       - image: circleci/node:10
     steps:
       - checkout
+      - run:
+          name: Check Tagged Push
+          command: |
+            TAG_VERSION=$(git tag --sort=-creatordate | head -n 1)
```

Comment:
Random question - why this and not something like `git describe --tags --abbrev=0`

Reply:
@dchai76, difference is easily visible when applied in our web repo